tools/manifest/manifest.py

```diff
@@ -91,7 +91,7 @@ def update(self, tree):
                     hash_changed = True
                 else:
                     new_type, manifest_items = old_type, self._data[old_type][rel_path]
-                if old_type == "reftest" and new_type != old_type:
+                if (old_type == "reftest" and new_type != old_type) or (old_type=="reftest_node" and new_type!=old_type):
```

Comment:
I think this can be written `old_type in ["reftest", "reftest_node"] and new_type != old_type`.

Reply:
Sorry, didn't think about that earlier. Made that change now. 
I'm not sure about how to write tests so I'll have to look into that

Reply:
It'd be more idiomatic to use a tuple (`("reftest", "reftest_node")`) here.